CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ### Fixes
 
+* **Fix api test** A test seemed to show that the api was not correctly applying the `strategy` kwarg when called, but it was a mistake with the file type and this verifies that the api is working as expected.
```

Comment:
please remove the CHANGELOG item. the policy is to only include changes that impact end users of the unstructured library in this file.

Reply:
Thanks. Wasn't positive on that.